### ISSUE TITLE
How/where do you want to manage documentation?

### ISSUE THREAD

**Issue by JayPanoz** [edrlab/readium-baseline#6]
Hi @HadrienGardeur and @llemeurfr 

Have just forked the baseline repo to [init reference](https://github.com/JayPanoz/readium-baseline/tree/master/Ref), which raises the question.

The ref contains UA + Readium 1 stylesheets, but I must still check the entire chromium’s source since the blink ones are quite old (last updated one year and a half ago). I guess this would probably bloat the Readium 2 repo and the baseline is where this kind of research can take place. But on the other hand, issues may have better traction in the former… 

I have some kind of introductory issue discussing goals, possible choices, pros and cons, etc. so it’s probably of interest to people currently working in the Readium 2 repo.

**Comment by HadrienGardeur** [comment]
I think we're going to create a new repo directly on the Readium organization.

Documentation should ideally be in Markdown in a specific folder of the repo, but it can also be in HTML and distributed using Github.io. HTML might me more relevant than usual in your case if you want to use CSS/JS directly in your docs.

**Comment by JayPanoz** [comment]
OK thanks. 

FYI, markdown sources were expected in any case, in addition to HTML (when HTML applicable).

**Comment by llemeurfr** [comment]
We discussed the name with Hadrien and talked about a simple "css" repo at the root of the readium organization space. But it may be weird for people forking the repo. After all "readium-css" may be better (Hadrien's take is that this wrok can be used outside the Readium-2 realm).
So, ok for "readium-css" ... or ?

**Comment by JayPanoz** [comment]
I can see why it could be weird in case of forking, name as you see fit.

Although the work can be used by others, we’ve started promoting this as readium 2 css. I guess some kind of consistency is at least expected. So it’s either changing readium 2 css or having everything use it?

**Comment by HadrienGardeur** [comment]
We have a similar case with the Readium Web Publication Manifest.

It's also a core part of Readium-2, but since it's not tied to it, we didn't called it Readium-2 Web Publication Manifest.

**Comment by JayPanoz** [comment]
Yeah, I can also see why using readium-2 in the name might let other implementers think the project is tied to R2, have had a quick discussion with one of them ± an hour ago. It makes sense to have something consistent at the global level.

Once again, feel free to name it as you see fit, it won’t make any difference as far as I’m concerned + you know the readium repo better than me.
